gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
 
-kotlin = "1.8.20" # should match Gradle's embedded Kotlin version https://docs.gradle.org/current/userguide/compatibility.html#kotlin
-kotlin-dokka = "1.8.20"
-kotlinx-serialization = "1.5.1"
+kotlin = "1.9.0" # should match Gradle's embedded Kotlin version https://docs.gradle.org/current/userguide/compatibility.html#kotlin
+kotlin-dokka = "1.9.0"
+kotlinx-serialization = "1.6.0"
 
 kotest = "5.6.2"
 
 gradlePlugin-android = "8.0.2"
```

Comment:
(The Android Gradle Plugin recently released [8.1.0/8.1.1 as part of the Giraffe stable release](https://developer.android.com/build/releases/gradle-plugin#8-1-0), not sure if it's worth updating in this PR)

Reply:
It's probably best to update AGP in another PR. I don't expect significant changes though, except to perhaps remove AGP classes that were deprecated and were removed in AGP8.